anywhere-fitness-fe/src/components/InstructorDashboard.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from "react";
+import axiosWithAuth from "../utils/axiosWithAuth";
+import ClassCard from "./ClassCard"
+import InstructorForm from './InstructorForm'
+
+const InstructorDashboard = () => {
+    const [allClasses, setAllClasses] = useState([]);
+    const [myClasses, setMyClasses] = useState([]);
+    const [refresh, setRefresh] = useState(false);
+    useEffect(() => {
+        axiosWithAuth()
+            .get("/instructor/classes")
+            .then((res) => {
+                console.log(res.data.data);
+                setMyClasses(res.data.data);
+            })
+            .catch((err) => {
+                console.log(err);
+            });
+    }, [refresh]);
+
```

Comment:
Oh. Did not have to look far to find another use case. Gotta stay DRY.
Shoot if more places use this logic with different url's you can pass it in the hook.

```
export const useFetchEndpoint = (url: string) => {
    // Define your state
    // Create an error state so you can present the user with a prompt if the API malfunctions or
    //     you miss an edge case
    // useEffect
    axios.get(url)
    // catch and log errors correctly. Include the url as well as the error description
}
// Reuse and refuse to write this again.
// Quality > Quantity
```